interface/wx/any.h

```diff
@@ -84,6 +84,24 @@ class wxAny
                  cannot be converted to a specific data type, wxAny will then
                  hold and manage reference to wxVariantData* similar to how
                  wxVariant does.
+
+                 Note that constructing from the list variant produces the
+                 memory leak if the list don't cleared explicitly. Don't fixed
+                 to not break the existing code.
```

Comment:
```suggestion
                 Note that objects constructed from list-valued variants
                 require the list to be explicitly cleared using `WX_CLEAR_LIST`
                 to avoid leaking memory. This unfortunate behaviour will not
                 be changed to prevent breaking the existing code relying on it.
```